cmd/podman/generate/systemd.go

```diff
@@ -85,6 +86,10 @@ func systemd(cmd *cobra.Command, args []string) error {
 		logrus.Warnln("The generated units should be placed on your remote system")
 	}
 
+	if !systemdOptions.New && systemdOptions.TemplateUnitFile {
+		return errors.New("--template option is relevant when --new is set")
```

Comment:
Couldn't we enforce `--new=true` when `--template` is set?  Maybe throw an error only if the user explicitly set `--new=false`.

I think that would be a bit more user friendly.

Reply:
I don't know enough to weigh in on @vrothberg's suggestion; but if you leave things as-is, could you change the wording to something like "--template option is **only** relevant when --new is set", or "--template requires --new"?